alibi/explainers/anchor_tabular.py

```diff
@@ -698,7 +705,7 @@ def __init__(self,
         self.ohe = ohe
         self.feature_names = feature_names
 
-        if ohe:
+        if ohe and categorical_names:
```

Comment:
We never validate the combinations that are consistent, this check is the easiest one to add for now.

Reply:
I don't think any errors are raised currently, we should definitely validate and do it. Perhaps a new issue.